specs/language/conversions.tex

```diff
@@ -130,6 +143,42 @@
 prvalue of type \texttt{matrix<T,x,y>}. The destination value is the source
 value replicated into each element of the destination.
 
+\p A glvalue of type \texttt{vector<T,1>} can be converted to a cxvalue of type
+\texttt{vector<T,x>} or a prvalue of type \texttt{vector<T,1>} can be converted
+to a prvalue of type \texttt{vector<T,x>}. The destination value is the value in
+the source vector replicated into each element of the destination.
+
+\Sec{Array and Class splat conversion}{Conv.asplat}
+
+\p A glvalue of type \texttt{T} can be converted to a cxvalue of type
+\texttt{struct S} or a prvalue of type \texttt{T} can be converted to a
+prvalue of type \texttt{struct S}, if there are valid conversions from
+\texttt{T} to each \texttt{U} in flattened \texttt{S}, see section
+\ref{Decl.init.Agg} for description of a flattened ordering. The destination
+value is the source value \texttt{T} replicated into each element of the
+flattened \texttt{S}.
+
+\p A glvalue of type \texttt{vector<T,1>} can be converted to a cxvalue of type
+\texttt{struct S} or a prvalue of type \texttt{vector<T,1>} can be converted to a
+prvalue of type \texttt{struct S}, if there are valid conversions from
+\texttt{T} to each \texttt{U} in flattened \texttt{S}. The destination value is
+the value in the source vector replicated into each element of the flattened
+\texttt{S}.
+
+\p A glvalue of type \texttt{T} can be converted to a cxvalue of type
+\texttt{U[]} or a prvalue of type \texttt{T} can be converted to a
+prvalue of type \texttt{U[]}, if there are valid conversions from
+\texttt{T} to each \texttt{V} in flattened \texttt{U}. The destination value
+is the source value \texttt{T} replicated into each element of the flattened
+\texttt{U[]}.
```

Comment:
`U[]` is not the same type as `U[N]`, so we should be clear we're talking about sized arrays here.